README.md

```diff
@@ -49,6 +49,10 @@ import uuid from "lib:uuid"; // (Note: exact import syntax is TBD).
 uuid(); // 52e6953d-edbe-4953-be2e-65ed3836b2f0
 ```
 
+All random values in UUIDs produced by this API must be generated from a
+**[cryptographically secure](https://en.wikipedia.org/wiki/Cryptographically_secure_pseudorandom_number_generator)**
```

Comment:
Maybe throw this link in the footer, as I could imagine us referencing it a few times throughout the document.

Certainly not a blocker for me though :smile: